src/silx/gui/widgets/OverlayMixIn.py

```diff
@@ -12,9 +12,28 @@ class OverlayMixIn:
     .. warning:: Any class inheriting from this mixin must also inherit from a QWidget.
     """
 
-    def __init__(self, parent):
+    def __init__(
+        self,
+        parent,
+        alignment: qt.Qt.AlignmentFlag = qt.Qt.AlignCenter,
+        alignment_offsets: tuple[int] = (0, 0),
+    ):
+        """
+        :param parent: parent widget
+        :param alignment: alignment of the overlay.
+        :param alignment_offsets: alignment offset as (horizontal offset, vertical offset). Values can be positive or negative. It will offset the alignment of this value
+        """
+        self._alignment: qt.Qt.AlignmentFlag = alignment
+        self._alignment_offsets: tuple[int] = alignment_offsets
         self._registerParent(parent=parent)
 
+    def getAlignment(self) -> qt.Qt.AlignmentFlag:
+        return self._alignment
+
+    def setAlignment(self, alignment: qt.Qt.AlignmentFlag):
+        self._alignment = alignment
+        self.repaint()
```

Comment:
`repaint` is synchronous, it's best to ask for an asynchronous repaint with:
```suggestion
        self.update()
```